Makefile.js

```diff
@@ -675,40 +673,12 @@ target.gensite = function(prereleaseVersion) {
                 const isFixable = rule && rule.meta.fixable;
                 const hasSuggestions = rule && rule.meta.hasSuggestions;
 
-                // Incorporate the special portion into the documentation content
-                const textSplit = text.split("\n");
-                const ruleHeading = textSplit[0];
-                const ruleDocsContent = textSplit.slice(1).join("\n");
+                text = text.replace("<!--FIXABLE-->", isFixable ? FIXABLE_TEXT : "")
+                    .replace("<!--SUGGESTIONS-->", hasSuggestions ? HAS_SUGGESTIONS_TEXT : "")
+                    .replace("<!--RECOMMENDED-->", isRecommended ? RECOMMENDED_TEXT : "");
```

Comment:
This information is already available in the rule's `meta`, so also depending on having a placeholder comment in the docs feels like an opportunity for us to forget to add one. Since the comments are always at the top of the text, could we insert the extra text at the top based solely on the rule's `meta` without looking for the comment?

Reply:
I was trying to make minimal changes to this logic for simplicity. This will all go away with the new docs site, so I don't want to spend a lot of time tweaking it.